libs/schema-validation/build.gradle.kts

```diff
@@ -35,6 +35,9 @@ dependencies {
     implementation ("com.google.code.gson:gson:2.10.1")
     implementation ("com.sun.activation:javax.activation:1.2.0")
 
+    implementation(project(":libs:commons-database"))
```

Comment:
What does the schema-validation need access to common-database for?  Can it be removed?

Reply:
Not needed. Removed.